ui/src/app/pages/admin/user/admin-user-profile.spec.tsx

```diff
@@ -707,34 +687,42 @@ describe('AdminUserProfile', () => {
         tierEligibilities,
       });
 
-      const wrapper = component();
-      expect(wrapper).toBeTruthy();
-      await waitOneTickAndUpdate(wrapper);
+      component();
+      await waitForNoSpinner();
 
-      const moduleTable = wrapper.find('[data-test-id="access-module-table"]');
-      expect(
-        findNodesContainingText(
-          moduleTable,
-          `requires eRA Commons for ${tiers} access`
-        ).exists()
-      ).toBeTruthy();
+      const table = screen.getByTestId('access-module-table');
+      within(table).getByText(`requires eRA Commons for ${tiers} access`, {
```

Comment:
while not strictly necessary, I believe the best practice is to still use an expect() here

Reply:
I agree. I wrestled with this for a while, but I think you're right, because it makes our assertions more explicit:
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-get-variants-as-assertions